Trajectory[] trajectory
std_msgs/Header header